routes/index.js

```diff
@@ -1,9 +1,13 @@
-var express = require('express');
+var express = require( 'express' );
 var router = express.Router();
 
 /* GET home page. */
-router.get('/', function(req, res, next) {
-  res.render('index', { title: 'Express' });
-});
+router.get( '/', function ( req, res ) {
+	var temp; 
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the declaration of the unused 'temp' variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://54.144.41.16/coding_rules#rule_key=javascript%3AUnusedVariable)